src/components/pages/Bookings/BookingsRecapTable/CellsFormatter/__specs__/BeneficiaryCell.spec.jsx

```diff
@@ -1,28 +1,29 @@
-import { shallow } from 'enzyme/build'
+import '@testing-library/jest-dom'
+import { render, screen } from '@testing-library/react'
 import React from 'react'
 
 import BeneficiaryCell from '../BeneficiaryCell'
 
-describe('components | BeneficiaryCell', () => {
-  it('should render a div with two span, one with firstname and lastname and the other the email address', () => {
+const renderBeneficiaryCell = props => {
+  return render(<BeneficiaryCell {...props} />)
+}
```

Comment:
```suggestion
const renderBeneficiaryCell = props => render(<BeneficiaryCell {...props} />)
```
Et ceux pour tous les autres.